config.yaml.example

```diff
@@ -19,7 +19,7 @@ install:
   #     size: 300
 
   # default partitions
-  # only 'oem', 'recovery', 'state' and 'persistent' objects allowed
+  # only 'bootloader', 'oem', 'recovery', 'state' and 'persistent' objects allowed
   # size in MiB
```

Comment:
Here is were we document all sizes are in megabytes. It is also documented in elemental-toolkit docs.